src/NuGet.Services.AzureSearch/SearchService/ISearchParametersBuilder.cs

```diff
@@ -8,9 +8,9 @@ namespace NuGet.Services.AzureSearch.SearchService
     public interface ISearchParametersBuilder
     {
         SearchParameters LastCommitTimestamp();
-        SearchParameters V2Search(V2SearchRequest request);
-        SearchParameters V3Search(V3SearchRequest request);
-        SearchParameters Autocomplete(AutocompleteRequest request);
+        SearchParameters V2Search(V2SearchRequest request, bool excludePackagesHiddenByDefault);
```

Comment:
These `excludePackagesHiddenByDefault` parameter names are a tad long, is there anything shorter? Maybe `isDefaultSearch` or something?